tests/everflow/test_everflow_ipv6.py

```diff
@@ -9,7 +9,7 @@
 from everflow_test_utilities import setup_info  # noqa: F401, E501 lgtm[py/unused-import] pylint: disable=import-error
 
 pytestmark = [
-    pytest.mark.topology("t0", "t1", "m0")
+    pytest.mark.topology("t0", "t1", "m0", "t0-52")
```

Comment:
t0-52 is included in topo t0, no need to add it specifically.

Reply:
addressed comment